janus_core/helpers/post_process.py

```diff
@@ -190,6 +190,7 @@ def compute_vaf(
     fft: bool = False,
     index: SliceLike = (0, None, 1),
     filter_atoms: MaybeSequence[MaybeSequence[Optional[int]]] = ((None),),
+    time_step: float = 1.0,
 ) -> NDArray[float64]:
```

Comment:
Not entirely new, but this is inconsistent with the corresponding docstring

```
    Returns
    -------
    MaybeSequence[NDArray[float64]]
        Computed VAF(s).
```

I think both need updating now that `vafs` is a tuple.